### DISCUSSION TITLE
latent couple + composable lora adaptation for COMFYUI

### DISCUSSION THREAD

**Discussion by manuiageek** [Comfy-Org/ComfyUI#1551] [General]
Hello, 
There is 2 extensions on automatic1111 that does composable lora (to be able to use 2 different lora character for example) and latent couple (to place on a mask defined where to place your character or stuff with a prompt).
The workflow is that when you define the mask for each things to place onto the image, you separate the prompts with the "AND" connector. It's not perfect but does the job (maybe I'm just not good enough with A1111 ... to be honest, I don't like it !)

Is there any chance to implement the same behaviour onto comfyui ? 

I really like how you can tweak all the weight and stuff with comfyui, I feel like I have more control over my final render.  It would be so cool to have that missing composable lora inside comfyui.

**Comment by alelordelo** [comment]
any news on this?

**Reply by manuiageek** [reply]
IPAdapter "might" be the solution
